lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,46 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Chrome normalizes background image style from getComputedStyle to be an absolute URL in quotes.
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const CSS_URL_REGEX = /^url\("([^"]+)"\)$/;
+  // Only find images that aren't specifically scaled
+  const CSS_SIZE_REGEX = /(auto|contain|cover)/;
+  const cssImages = [...document.querySelectorAll('*')].reduce((images, element) => {
+    const style = window.getComputedStyle(element);
+    if (!CSS_URL_REGEX.test(style.backgroundImage) ||
+        !CSS_SIZE_REGEX.test(style.backgroundSize)) {
+      return images;
+    }
+
+    const imageMatch = style.backgroundImage.match(CSS_URL_REGEX);
+    const url = imageMatch[1];
+
+    // Heuristic to filter out sprite sheets
+    const differentImages = images.filter(image => image.src !== url);
+    if (images.length - differentImages.length > 2) {
+      return differentImages;
+    }
+
+    images.push({
+      src: url,
+      clientWidth: element.clientWidth,
+      clientHeight: element.clientHeight,
+      naturalWidth: Number.MAX_VALUE,
+      naturalHeight: Number.MAX_VALUE,
```

Comment:
Is this b/c "css images have no natural size"? Might want to add that comment here too. I was expecting these to be `element.clientWidth/Height`.